exercises/robot-simulator/src/example/java/GridPosition.java

```diff
@@ -9,21 +9,26 @@
         this.y = y;
     }
 
-    /*
-     * This equals method is of deliberately narrow scope (only allows comparison with another GridPosition) to increase
-     * readability. In general, one should provide a full implementation of Object.equals(Object obj) and a
-     * corresponding implementation of Object.hashCode(). See
-     *
-     * https://docs.oracle.com/javase/7/docs/api/java/lang/Object.html#equals(java.lang.Object)
-     *
-     * and
-     *
-     * https://docs.oracle.com/javase/7/docs/api/java/lang/Object.html#hashCode()
-     *
-     * for more information.
-     */
-    boolean equals(final GridPosition gridPosition) {
-        return this.x == gridPosition.x && this.y == gridPosition.y;
-    }
+    @Override
```

Comment:
The spacing seems to have gone a bit wrong here and in the test file. It should be indented with four spaces :) If you're using IntelliJ you can use `ctrl + alt + L` to automatically format the code (`cmd + alt + L` on a mac). That should fix spacing issues in the current file and will even remove unused imports if you tell it to in the reformat code dialog box (see https://www.jetbrains.com/help/idea/reformatting-source-code.html). On eclipse it's `ctrl + shift + F` :)